crates/bevy_asset/src/server/mod.rs

```diff
@@ -1500,8 +1496,35 @@ pub enum LoadState {
     Failed(Arc<AssetLoadError>),
 }
 
+impl LoadState {
+    /// Returns `true` if this instance is [`LoadState::Loading`]
+    pub fn is_loading(&self) -> bool {
+        matches!(self, Self::Loading)
```

Comment:
I could prefer this over the matches in the tests

Reply:
Yes.